src/common/hooks/useFilters.tsx

```diff
@@ -61,6 +61,10 @@ export default function useFilters({
     filter = hasPagination ? { page: 1, limit, ...filter } : filter;
     setQueryParams(Object.assign({}, qParams, filter), { replace: true });
   };
+  const updateSerchByField = (filter: FilterState) => {
+    filter = hasPagination ? { page: 1, limit, ...filter } : filter;
+    setQueryParams(Object.assign({}, filter), { replace: true });
+  };
```

Comment:
It is doable without needing to make this change.

Also this causes issue when used along with advanced filters. The applied advanced filters gets unapplied when we switch the search criteria (phone number to name for example).

Use `removeFilters` in conjunction with `updateQuery` instead of implementing this.

Reply:
@rithviknishad ,got it ,will fix the issue

Reply:
done